api.yaml

```diff
@@ -198,6 +198,64 @@ paths:
               "time": "#!index .Request.Query.time 0!#"
             }
           success: '{"status": 200, "bodyPassthrough": true}'
+          error: >
+            {
+              "status":
+              #! if eq .Response.Body.errorType "InvalidInput" !# 400,
+              #! else !#
+              #! if eq .Response.Body.errorType "NotFound" !# 404,
+              #! else !#
+              #! if eq .Response.Body.errorType "PartitionConflict" !# 409,
```

Comment:
Task: this is in the wrong lambda section, should be down under `/ops/pgsql/v1/partitions`

Reply:
Oh dang. thanks for catching that